src/libcharon/daemon.h

```diff
@@ -203,7 +203,7 @@ typedef struct daemon_t daemon_t;
 /**
  * Number of threads in the thread pool, if not specified in config.
  */
-#define DEFAULT_THREADS 16
+#define DEFAULT_THREADS 8
```

Comment:
Thinking about it again, it's better to change the number of threads from the startup configuration of the users of SL instead of changing it in the code:
```
/usr/bin/sed -i 's/# threads = 16/threads = 8/' /etc/strongswan.d/charon.conf
```